powersimdata/input/tests/test_export_to_pypsa.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 26 10:15:01 2022
+
+@author: fabian
+"""
+import pytest
+
+from powersimdata.input.export_data import export_to_pypsa, pypsa_available
+from powersimdata.input.grid import Grid
+
+
+def assert_columns_preserved(n):
+    assert "Vmax" in n.buses
+    assert "ramp_10" in n.generators
+    assert "rateB" in n.lines
+    assert "QminF" in n.links
+
+
+def assert_columns_deleted(n):
+    assert "Vmax" not in n.buses
+    assert "ramp_10" not in n.generators
+    assert "rateB" not in n.lines
+    assert "QminF" not in n.links
+
+
+@pytest.mark.skipif(not pypsa_available, reason="Package PyPSA not available.")
+def test_export_grid_to_pypsa():
+    grid = Grid("USA")
+
+    n = export_to_pypsa(grid)
```

Comment:
Are there any tests in PyPSA for whether a Network object is internally consistent, without trying to instantiate the optimization problem itself? E.g. that branch endpoints are present in the bus table, etc.

Reply:
Unfortunately there are no dedicated tests. However some tests are run when the components are added to the network. For example when adding lines, it check that all `bus0` and `bus1` entries are existent buses in the network.